t/dummy-isotovideo.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo "dummy isotovideo started with arguments: $@"
+echo "arguments: $@" > autoinst-log.txt
+echo '{"current_test":"","status":"finished","test_execution_paused":0}' > autoinst-status.json
```

Comment:
Interesting. What do we need this for?

Reply:
I guess not. I'm still experimenting. I'll also look into the other comments.

Reply:
These `echo`s are used to fake test results. The web UI requires at least one passed test module to be present to consider a job passed and `autoinst-status.json` is used by the worker upload to determine the test status.